docs/protocol/life_cycle_of_a_service.md

```diff
@@ -0,0 +1,41 @@
+The life cycle of a service registered in the Autonolas Protocol passes through the following states:
+
+* pre-registration,
+* active registration,
+* finished registration,
+* deployed, and
+* terminated bonded.
+
+You can see the current state of a service by browsing the [services section](https://protocol.autonolas.network/services) in the frontend, and clicking on the corresponding service.
+
+<figure markdown>
+![Life cycle of a service](images/life_cycle_of_a_service.svg){ align=left width=600 }
+</figure>
+
+Only the service owner can change the state of a given service. Connect your wallet to the frontend in order to make changes.
+
+## Pre-registration
+
+A service that [has just been registered](./register_packages_on-chain.md#register-a-service) will be in pre-registration state.
+In this state the service owner can make amendments by pressing the button _Update_.
+Press the button _Activate Registration_ to advance to the next state. The connected wallet will ask to approve the transaction.
+
+## Active registration
+
+Agent operators must specify the addresses (starting by `0x...`) of their agent instances that are part of the service, and press _Register Agents_ to associate them with the service.
+
+Once all agent instances have been registered, the service owner can press the button _Terminate_ to proceed to the next state. Wallets will ask to approve the transactions.
```

Comment:
This sentence must be moved to the Finished-registration section. Here we could talk about _Terminate_ if the owner does not want to deploy the service, and thus all the activity is terminated and service goes into the Terminated-bonded state.